modules/apps/dynamic-data-mapping/dynamic-data-mapping-uad-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/uad/anonymizer/test/DDMFormInstanceRecordUADAnonymizerTest.java

```diff
@@ -105,7 +104,7 @@ protected boolean isBaseModelAutoAnonymized(long baseModelPK, User user)
 		throws Exception {
 
 		DDMFormInstanceRecord ddmFormInstanceRecord =
-			_ddmFormInstanceRecordLocalService.getDDMFormInstanceRecord(
+			DDMFormInstanceRecordLocalServiceUtil.getDDMFormInstanceRecord(
```

Comment:
@marcosapmf, you can keep the `_ddmFormInstanceRecordLocalService` on the tests. My goal with my previous review was ask you to not use the local services as a parameter of your util classes. Can you please revert this change and all others similar to it and re-send?

Thanks.